services/validation.tsx

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { getCookie } from 'cookies-next';
+
+export async function verifyToken() {
+    const token = getCookie('access_token');
+    const url = `${process.env.NEXT_PUBLIC_AF_PORTAL_BACKEND_URL}/verify`;
+
+    if (!token) {
+        return { isValid: false, message: 'Token not found' };
+    }
+
+    try {
+        const response = await axios.get(url, {
+            headers: {
+                Authorization: `Bearer ${token}`,
+            },
+        });
+        return { isValid: true, data: response.data };
+        
+    } catch (error) {
+        return { isValid: false, message: 'Token invalid' };
```

Comment:
Did you try the API calls? Can we get more detailed errors from the call?

Reply:
Sure